components/button/StartButton.tsx

```diff
@@ -0,0 +1,16 @@
+import styles from "./StartButton.module.css";
+
+function StartButton() {
+  function handleClick(e: { preventDefault: () => void }) {
+    e.preventDefault();
+  }
+
+  return (
+    <a href="#" onClick={handleClick}>
+      <button className={styles.start}>
+        <div className={styles.text}>Start</div>
+      </button>
+    </a>
```

Comment:
You can be safer if you give props to the functions. With this you have more security and you will be suggested what the function needs

```suggestion
export type StartButtonProps = {
 label: string
 handleClick: MouseEventHandler<HTMLButtonElement>
}

function StartButton({ label, handleClick }: StartButtonProps) {
  function handleClick(e: { preventDefault: () => void }) {
    e.preventDefault();
  }

  return (
      <button className={styles.start} onClick={handleClick}>
        <div className={styles.text}>{label}</div>
      </button>
```